### PR TITLE
UI Overhaul: Add delete button on projects settings page

### PR DESCRIPTION
**Pull request by mausworks**
## Description

This PR does what it says on the tin:
image

## Checklist

- [x] I have manually tested my changes and I am happy with the result.

### CODE REVIEW

File: services/orchest-webserver/app/app/views/views.py [hunk 1]
```diff
@@ -579,12 +579,12 @@ def projects_post():
...
 
-    @app.route("/async/projects", methods=["DELETE"])
-    def projects_delete():
+    @app.route("/async/projects/<project_uuid>", methods=["DELETE"])
+    def projects_delete(project_uuid):
 
         try:
             with TwoPhaseExecutor(db.session) as tpe:
-                DeleteProject(tpe).transaction(request.json["project_uuid"])
+                DeleteProject(tpe).transaction(project_uuid)
```
Comment (mausworks) [inline services/orchest-webserver/app/app/views/views.py:587]: Using a body in a DELETE request is non-standard. When I moved all API calls to /api I thought I might as well fix this.

**Review by iannbing — APPROVED** [review]
LGTM!

**Comment by mausworks** [comment]
> Looks good! What about a slightly less fluorescent error color? E.g. #d32f2f from (Mui-error default) (https://mui.com/material-ui/customization/palette/#default-values).

I agree that the red is a bit too red. But it's the same error-color we use for everything else. I'm definitely for changing it to a darker one in the future though. :+1:

**Comment by mausworks** [comment]
@ricklamers I [pushed a commit](https://github.com/orchest/orchest/pull/1060/commits/5a33fcc0970e56c6aecc8bc77e49913ad66efe42) to the [feat/ui-overhaul](https://github.com/orchest/orchest/pull/1060) branch that resets the red error color to #d32f2f :)

**Pull request merged**